charts/backstage/README.md

```diff
@@ -1,7 +1,6 @@
-
```

Comment:
This is too big of a change. Please run helm docs via pre-commit or with the following parameters:

https://github.com/backstage/charts/blob/d3fc1b8509d2389c3099f2ff2036325b9dc2d90c/.pre-commit-config.yaml#L7-L12

Reply:
Was already asking myself what I did wrong. Fixed it!